report_thesis/src/sections/introduction.tex

```diff
@@ -0,0 +1,45 @@
+This plasma plume emits light that is captured through three distinct spectrometers to collect a series of spectral readings.
+These spectra consist of various emission lines, each associated with a specific element, and their intensity reflects the concentration of that element in the sample.
+Consequently, a spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological formations.\cite{cleggRecalibrationMarsScience2017}
+
+Analyzing \gls{libs} data is computationally challenging due to high multicollinearity within spectral data, which diminishes the effectiveness of traditional linear analysis.
+The multicollinearity, which stems from correlations among spectral channels and elemental emission characteristics, complicates data interpretation.
+Additionally, there are multiple interacting physical processes
+Additionally, \textit{matrix effects} in \gls{libs} spectra arise when various physical interactions cause emission line intensities to change without a corresponding shift in the element's actual concentration
+This phenomenon introduces variability that complicates the straightforward interpretation of spectral data, challenging the accuracy of computational models tasked with determining elemental composition.\cite{andersonImprovedAccuracyQuantitative2017}
+
+For analyzing Martian geological samples, the \gls{chemcam} team currently uses the \gls{moc} model\cite{cleggRecalibrationMarsScience2017}.
+This model integrates \gls{pls} and \gls{ica} to predict the composition of major oxides.
+Though the MOC model has proven useful, it suffers from limitations in predictive accuracy and robustness.
+In \citet{p9_paper}, we created a replica of the MOC model and identified which components were responsible for these limitations.
+Through a series comparative experiments, we showed that the model selection was the primary cause of these limitations, and we showed how both \gls{ann} and \gls{gbr} methods could be used to improve the model's predictive accuracy and robustness.
+
+This is further underscored by work from the SuperCam team.
+In 2021, the Perseverance rover landed on Mars, equipped with the SuperCam instrument, which is the successor to the \gls{chemcam} instrument.
+As part of the ongoing work to support the SuperCam instrument, \citet{andersonPostlandingMajorElement2022} experimented with various machine learning models to predict the composition of major oxides in geological samples using the SuperCam \gls{libs} calibration dataset.
+While the team decided to retain \gls{pls} for analyzing certain oxides, \gls{ica} was entirely discontinued.
+Instead, models based on \gls{gbr}, \gls{rf}, and \gls{lasso} were selected for other oxides.
```

Comment:
They included a few other methods as well, e.g. Elastic Net. Do we want to be exhaustive here? Otherwise, we can just say `among others`.